src/main/java/forestry/Forestry.java

```diff
@@ -199,12 +201,16 @@ public void setupClient(FMLClientSetupEvent event) {
 	}
 
 	private void gatherData(GatherDataEvent event) {
+		CapabilityFluidHandler.register();	//TODO test
```

Comment:
Forge calls this event in preInit which is either too late or not called on the data run. So to be able to create recipes with containers I added a call here. Probably forge should fix this.